platform/ios/CHANGELOG.md

```diff
@@ -4,15 +4,35 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 ## master
 
+### Networking and storage
+
+* Added a new `MGLMapSnapshotter` class for capturing rendered map images from an `MGLMapView`’s camera. ([#9891](https://github.com/mapbox/mapbox-gl-native/pull/9891))
+* Reduced the time it takes to create new `MGLMapView` instances in some cases. ([#9864](https://github.com/mapbox/mapbox-gl-native/pull/9864))
+* Added support for forced cache revalidation that will eliminate flickering that was sometimes visible for certain types of tiles (i.e., traffic tiles). ([#9670](https://github.com/mapbox/mapbox-gl-native/pull/9670), [#9103](https://github.com/mapbox/mapbox-gl-native/issues/9103))
```

Comment:
Grammar nitpick: if this affects other types of tiles beyond traffic, we should use “e.g.” instead of “i.e.” — the former indicates one possible example of a thing, while the latter indicates it is the only example of the thing.